moveit_core/utils/include/moveit/utils/moveit_status.h

```diff
@@ -32,35 +32,31 @@
  *  POSSIBILITY OF SUCH DAMAGE.
  *********************************************************************/
 
-/* Author: Ioan Sucan
- * Desc: Empty adapter that just calls the planner
- */
+#pragma once
 
-#include <moveit/planning_request_adapter/planning_request_adapter.h>
-#include <class_loader/class_loader.hpp>
+#include <string>
 
-namespace default_planner_request_adapters
+namespace moveit
 {
-/** @brief Empty adapter that just calls the planner. */
-class Empty : public planning_request_adapter::PlanningRequestAdapter
```

Comment:
This diff doesn't make much sense since this is a new file with completely different content :shrug:. We could consider using this MoveItStatus in other places as well and/or add it to the motion plan response since it is more verbose than a plain error code.